src/Componets/MainCrad/ProductCard.js

```diff
@@ -1,24 +1,21 @@
+import "./product.css";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import { useFilterContext } from "../../Contexts/filter-context";
+import { useCart } from "../../Contexts/cartContext";
+import { Link } from "react-router-dom";
+import { useWishlistContext } from "../../Contexts/wishListContext";
```

Comment:
Keep the contexts in one index file and import them from there to reduce the lines of code.